### DISCUSSION TITLE
[Feature] Tooltips with latex

### DISCUSSION THREAD

**Discussion by SPyofgame** [squidfunk/mkdocs-material#5231] [Ideas]
I would like to inquire if it is possible to incorporate some advanced LaTeX formatting, such as mathematical equations and computer science notation, to enhance the quality of my content. For example, hinting my viewers whenever they hover over a weirdly defined symbol. Is there a way to achieve this without relying on the use of annotations or similar techniques?

![image](https://user-images.githubusercontent.com/62207095/226241476-36f7ee75-7078-4796-86e4-399b330d5d81.png)

```cpp
[Bitmask][Reference]

[Reference]: https://github.com/ "Bitmask use bit $0, 1$ to represent a set of number, where integer $p$ is included in the set when and only when the $p$ bit of the bitmask is set ($1$)."
```

For something like that, can I use latex?

And by the way, may I ask that are you and [squidfunk](https://github.com/squidfunk) both working on the same pymdown project ? 

**Note:** [Facelessuser discussion](https://github.com/facelessuser/pymdown-extensions/discussions/1993)

**Comment by squidfunk** [comment]
This is currently not possible, as `title` cannot (and should not) contain complex markup. This is necessary to keep compatibility with screen readers that don't expect `title` tags to contain markup, as they will read the title out loud. If JavaScript fails to load for some reason, this functionality will be entirely broken.

We're evaluating other possibilities to add tooltips and arbitrary markup at arbitrary positions, like [instant previews](https://twitter.com/squidfunk/status/1466794654213492743?lang=en) and a glossary functionality. In the meantime, [annotations](https://squidfunk.github.io/mkdocs-material/reference/annotations/) can already be used and should be a good replacement. 😊

**Reply by SPyofgame** [reply]
lmao I was about to ask if there is possible to make the text even richer and that [instant previews] surely blowing my mind 👀

**Reply by squidfunk** [reply]
Awesome! I hope to be able to set some time aside to work on this later this year.

**Comment by squidfunk** [comment]
> And by the way, may I ask that are you and [squidfunk](https://github.com/squidfunk) both working on the same pymdown project ?

Seconding facelessuser's answer here: https://github.com/facelessuser/pymdown-extensions/discussions/1993#discussioncomment-5363941
